samples/tests/package.json

```diff
@@ -19,7 +19,6 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@azure/communication-react": "1.3.1-beta.1",
```

Comment:
These tests currently don't use this dependency at all, so dropping it (instead of adding to CI time by building all of `@azure/communication-react` before running these tests.

Let's see what CI has to say about this ;)